src/tools/build-manifest/src/main.rs

```diff
@@ -287,28 +283,10 @@ impl Builder {
     }
 
     fn add_packages_to(&mut self, manifest: &mut Manifest) {
-        macro_rules! package {
-            ($name:expr, $targets:expr) => {
-                self.package($name, &mut manifest.pkg, $targets, &[])
-            };
+        for pkg in PkgType::all() {
+            let fallback = if pkg.use_docs_fallback() { DOCS_FALLBACK } else { &[] };
```

Comment:
It seems like this might as well go into `self.package(...)`, now that it's just a call to a method on pkg?

Reply:
Good idea, done.